src/main/java/seedu/address/model/tag/TutorialTag.java

```diff
@@ -17,4 +17,8 @@ public TutorialTag(String tagName, TagStatus tagStatus) {
     public boolean isSameTutorialTag(TutorialTag tutorialTag) {
         return this.getTagName().equals(tutorialTag.getTagName());
     }
+    @Override
+    public String toString() {
+        return getTagName();
+    }
```

Comment:
Not very safe. Can just make the caller call super.getTagName()

Reply:
Sure sure will update in my next PR